src/test/java/com/ncc/aif/ScalingTest.java

```diff
@@ -77,20 +77,64 @@
     private int entityIndex = 1;
     private int eventIndex = 1;
     private int assertionIndex = 1;
+    private int clusterCount = 0;
 
-    private static final String NAMESPACE = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/SeedlingOntology";
+    private static final String NAMESPACE = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/LDCOntology";
+    private static final boolean ALLOW_NUMERIC_TYPES = false;
 
     private final Random r = new Random();
     private List<Resource> entityResourceList = null;
 
-    private final String filename = "scalingdata.ttl";
+    private String filename = "scalingdata-" + entityCount + "x" + eventCount + ".ttl";
 
     // Whether to use in memory or disk based.
     // Note:  TDB2 requires transactions, which we do not do!  Do not use it!
     private enum MODEL_TYPE {
         MEMORY, TDB
     }
 
+    // These NIST restrictions correspond to (and should be updated from) com/ncc/aif/ExamplesAndValidationTest.java.
+    private enum NIST_RESTRICTION {
```

Comment:
It seems to me like you're building an enum with values for the frequency for violating restrictions. Would it make sense to just include these values in the enum? https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Reply:
Cool, didn't know enums could do that.  I enhanced the Enum and removed the constants and map.